### ISSUE TITLE
[Accessibility]A11y_Java Tooling for IntelliJ/Eclipse - New Features_Application Graph_Color Contrast: Color contrast ratio for Tab items & links (blue color text) is 3.6:1 which is < 4.5:1

### ISSUE THREAD

**Issue by Tanushagottipati** [microsoft/azure-tools-for-java#3566]
[Check out Accessibility Insights! ](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faccessibilityinsights.io%2F&data=02%7C01%7Cv-tagott%40microsoft.com%7Cb67b2c4b646d4f9561a208d6f4b5c39b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636965458847261333&sdata=PJMiKjb64uwhBN98LZpctnBiAVeOvw%2BLBrnxBtrjl2M%3D&reserved=0)- Identify accessibility bugs before check-in and make bug fixing faster and easier

GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_JavaToolingforIntelliJ/Eclipse–NewFeature; #MAS1.4.3; #ColorContrast; #C+E-H; #JavatoolingforIntelliJ/Eclipse

**Environment Details:**
Eclipse IDE for Enterprise Java Developers.
Version: 2019-06 (4.12.0)
Build id: 20190614-1200
Solution Installed: The Azure Toolkit for Eclipse 3.17.0

**OS Details:** 
OS: Windows (Version 1903, Build: 18362.175)
MAS Violated: MAS1.4.3

**Repro Steps:** 
1) Open Java EE - Eclipse application
2) Install 'Azure Toolkit for Eclipse' from Marketplace
3) Open Eclipse and install the Scala plug-in 
4) Tab till tools-->Azure-->Azure Sign in and sign into Azure account and select any subscription
5) In Azure Explorer section, tab till HDInsight and expand it. Select 'Spark on HDInsight (Scala)'. Tab till Next and press enter
6) Tab till Next button and press enter. Press Finish button
7) In Project Explorer, Expand the created Scala project right-click src, New-->Other and press enter
8) Tab till Scala Wizards, expand and select 'Scala Object' and press next
9) Enter the name of the file and press Finish button
10) Open Scala file and write any code in the editor
11) In Project Explorer view, right-click your project, choose Azure-->Submit Spark Application to HDInsight
12) Enter all the required details and press submit button
13) In Azure Explorer tab, tab till HDInsight-->Cluster-->Job and press enter
14) Select any one the application and press enter. 
15)Verify the color contrast ratio for tab items(Application Graph, Stage Summary, Executors, Task Summary, Logs) & links(Open spark history UI, open yarn UI) is >= 4.5:1 or not

**Actual Result:** 
Color contrast ratio for Tab items & links (blue color text) is 3.6:1 which is < 4.5:1

**Expected Result:** 
Color contrast ratio for Tab items & links (blue color text) should be >= 4.5:1

**User Impact:** 
Low vision users will find difficult to read the content if the text does not meet the standard color contrast ratio

**MAS Reference:** https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}

![Bug_14_Color Contrast for Tab items and links is less than 4 51](https://user-images.githubusercontent.com/41109135/65323956-60791180-dbc8-11e9-95de-8c7fda922f51.png)

**Comment by t-rufang** [comment]
The job view page is wrote as static web pages with the help of JavaFX WebView which has no support for Accessibility. Please refer to JDK issue [JDK-8177382](https://bugs.openjdk.java.net/browse/JDK-8177382)

**Comment by Alexandria-R** [comment]
This has been approved to close by SME.

**Comment by Tanushagottipati** [comment]
Closing this issue based on above comments

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749677
